src/citra/citra.cpp

```diff
@@ -391,6 +391,8 @@ int main(int argc, char** argv) {
         return -1;
     case Core::System::ResultStatus::Success:
         break; // Expected case
+    default:
+        LOG_ERROR(Frontend, "Error while loading ROM: {}", system.GetStatusDetails());
```

Comment:
```suggestion
        LOG_ERROR(Frontend, "Error while loading ROM: {}", system.GetStatusDetails());
        break;
```
It's convention to break in default too. Ditto for the other switch as well.

Reply:
Done.

Reply:
Thanks!